### PR TITLE
Announcement for influxdb slack channel

### PR DESCRIPTION
**Pull request by corylanou**
/cc @otoolep

### CODE REVIEW

File: content/blog/2015/11/13/influxdb_slack_channel.md [hunk 1]
```diff
@@ -0,0 +1,21 @@
...
+date: 2015-11-13
+publishdate: 2015-11-13
+---
+
+
+With the recent blog on [contributing to InfluxDB](https://influxdb.com/blog/2015/11/05/start_contributing.html), we have seen a significant increase in community pull requests!  To make sure that we are supporting the community as much as possible, we will now be active on the [#influxdb](https://gophers.slack.com/messages/influxdb/) channel on [Gopher Slack Community](https://gophers.slack.com/messages/general/).
+
+## Why not create your own community slack?
+
+The Gopher Slack community is very vibrant.  Since InfluxDB is a Go project, it makes sense to stay within that community.  It also allows developers who want to learn Go as well as contribute to projects like InfluxDB the perfect ecosystem to ask questions about both Go and InfluxDB.
```
Comment (otoolep) [inline content/blog/2015/11/13/influxdb_slack_channel.md]: " It also allows developers who want to learn Go as well as contribute to projects like InfluxDB the perfect ecosystem to ask questions about both Go and InfluxDB.." - doesn't read quite right. Do you mean:

"It also allows developers who want to learn Go as well as contribute to projects like InfluxDB, making it the perfect ecosystem to ask questions about both Go and InfluxDB."

**Comment by otoolep** [comment]
Looks good to me, with one suggested correction.

File: content/blog/2015/11/13/influxdb_slack_channel.md (hunk 1 above)
Comment (corylanou) [inline content/blog/2015/11/13/influxdb_slack_channel.md]: Much better.  Updated.  Thanks!

**Comment by gunnaraasen** [comment]
+1

**Comment by gunnaraasen** [comment]
@corylanou you can merge this now. The post won't actually be generated by hugo until the publish date.

**Pull request merged**